sys/include/usb/usbus/msc.h

```diff
@@ -50,17 +50,6 @@ extern "C" {
  */
 #define USBUS_MSC_EP_OUT_REQUIRED_NUMOF  1
 
-/**
- * @brief USBUS MSC Number of exported MTD device through USB
- */
-#ifndef USBUS_MSC_EXPORTED_NUMOF
```

Comment:
Maybe `USBUS_MSC_EXPORTED_NUMOF` should not be dropped as not every MTD  in `MTD_NUMOF` will be a USB MTD.

Reply:
```C
    if (CONFIG_USBUS_MSC_AUTO_MTD) {
        for (unsigned i = 0; i < MTD_NUMOF; i++) {
            usbus_msc_add_lun(usbus, mtd_default_get_dev(i));
        }
    }
```

Or is it 🤔